File: src/test/java/FileParserTest.java
import org.junit.Test;

import java.util.List;

/**
 * Author: stk
 * Date: 18/4/2
 */
public class FileParserTest {
    @Test
    public void parseFile() {
        String file = "C:\\Users\\stk\\Documents\\Docs\\Papers\\EASE2018Log\\Data\\elasticsearch\\test\\framework\\src\\main\\java\\org\\elasticsearch\\transport\\MockTcpTransport.java";
        List<String> result = FileParser.parseFile(file);
        assert result != null;
        result.forEach(System.out::println);
    }
}

File: src/test/scala/FileParserTest.scala
import java.io.File

import org.junit.Test

/**
  * Author: stk
  * Date: 18/4/6
  */
class FileParserTest {
  val path = "C:\\Users\\stk\\Downloads\\test-data\\transport\\nio\\MockNioTransport.txt"
  val path1 = "C:\\Users\\stk\\Downloads\\test-data\\transport\\AbstractSimpleTransportTestCase.txt"

  @Test
  def parseFile(): Unit = {
    FileParser.parseFile(new File(path1))
  }
}
